demo/src/app/components/shared/api-docs/api-docs-class.component.ts

```diff
@@ -19,13 +22,30 @@ import {Analytics} from '../../../shared/analytics/analytics';
     .content-cell {
       width: 75%;
     }
+    .fragment {
+      display: none;
+      font-size: 1em;
+      line-height: inherit;
+      position: absolute;
+      margin-left: -1.2em;
+      padding-right: 0.5em;
+    }
+    h2:hover .fragment {
+      display: inline-block;
+    }
     `
   ]
 })
-export class NgbdApiDocsClass {
+export class NgbdApiDocsClass implements AfterViewInit, OnDestroy {
   apiDocs: ClassDesc;
+  routeFragmentSubscription: Subscription;
+  scrollToFragment = scrollToFragment;
```

Comment:
The `click` event calling it was causing an error in the template file, but it was actually unneeded.